tutorial/requestcredential.js

```diff
@@ -4,13 +4,14 @@ var uport = require('../lib/index.js');
 var jsontokens = require('jsontokens')
 var bodyParser = require('body-parser')
 
-var signer = uport.SimpleSigner('4d8f5b6ba9298038b6cde46e5fa36d7b7a5846cdcd1c63ad9f5dea4384f9e650')
+var signer = uport.SimpleSigner('5acea265dcbf01355956b36f82793a13caf1be650bb74ca7a40da74b412d44b5')
 
 
 var credentials = new uport.Credentials({
   appName: 'Credential Tutorial',
-  address: '0x982024e3540fb5ba26b7e169dcd98503f10686f5',
-  signer: signer
+  address: '2opiNrGSxTW6sFo6ervHZnEeEXGNGAoupFN',
+  signer: signer,
+  networks: {'0x4': {'registry' : '0x2cc31912b2b0f3075a87b3640923d45a26cef3ee', 'rpcUrl' : 'https://rinkeby.infura.io'}}
```

Comment:
@christianlundkvist where does it say that? In the library? The app?

Try running `yarn` it may be your dependencies aren't up to date